ResourceGenerator/Module.Template.psm1

```diff
@@ -395,38 +381,38 @@ function Test-TargetResource
     Add-M365DSCTelemetryEvent -Data $data
     #endregion
 
-    Write-Verbose -Message "Testing configuration of {$<PrimaryKey>}"
+    Write-Verbose -Message 'Testing configuration of {$<PrimaryKey>}'
```

Comment:
This change will break the string output of the variable after the replacement. There need to be proper quotes for this string.

Reply:
Done